app/controllers/articles_controller.rb

```diff
@@ -0,0 +1,24 @@
+class ArticlesController < ApplicationController
+  def new
+    @article = Article.new
+  end
+
+  def create
+    @article = Article.new(article_params)
+    if @article.save
```

Comment:
It would be great to add a new line between these two lines. It will make the code much more readable. We have to remember that the code isn't only for us now. It's also for our future selves. We should be kind to them.